docs/content/preview/yugabyte-platform/configure-yugabyte-platform/alert-policy-templates.md

```diff
@@ -131,39 +131,77 @@ sum by (universe_uuid) (ybp_health_check_node_master_fatal_logs{universe_uuid="$
+  count by (node_prefix) (
+        label_replace(
+          max_over_time(up{export_type=~"master_export|tserver_export",node_prefix="$node_prefix"}[15m]),
+          "exported_instance",
+          "$1",
+          "instance",
+          "(.*)"
+        )
+      <
+        1
+    and on (node_prefix, export_type, exported_instance)
+      (min_over_time(ybp_universe_node_function{node_prefix="$node_prefix"}[15m]) == 1)
+  )
+>
+  0
 ```
 
 #### DB instance restart
 
 Universe `'$universe_name'` Master or T-Server has restarted `$value` times during last 30 minutes.
 
 ```promql
-max by (node_prefix) (changes(yb_node_boot_time{node_prefix="$node_prefix"}[30m]) and on (node_prefix) (max_over_time(ybp_universe_update_in_progress{node_prefix="$node_prefix"}[31m]) == 0)) > 0
+  max by (node_prefix) (
+      changes(yb_node_boot_time{node_prefix="$node_prefix"}[30m])
+    and on (node_prefix)
+      (max_over_time(ybp_universe_update_in_progress{node_prefix="$node_prefix"}[31m]) == 0)
+  )
+>
+  0
 ```
 
 #### DB queues overflow
 
 Database queues overflow has been detected for universe `'$universe_name'`.
 
 ```promql
-sum by (node_prefix) (increase(rpcs_queue_overflow{node_prefix="$node_prefix"}[10m])) + sum by (node_prefix) (increase(rpcs_timed_out_in_queue{node_prefix="$node_prefix"}[10m])) > 1
+    sum by (node_prefix) (increase(rpcs_queue_overflow{node_prefix="$node_prefix"}[10m]))
+  +
+    sum by (node_prefix) (increase(rpcs_timed_out_in_queue{node_prefix="$node_prefix"}[10m]))
+>
+  1
 ```
 
 #### DB memory overload
 
 Database memory rejections have been detected for universe `'$universe_name'`.
 
 ```promql
-sum by (node_prefix) (increase(leader_memory_pressure_rejections{node_prefix="$node_prefix"}[10m])) + sum by (node_prefix) (increase(follower_memory_pressure_rejections{node_prefix="$node_prefix"}[10m])) + sum by (node_prefix) (increase(operation_memory_pressure_rejections{node_prefix="$node_prefix"}[10m])) > 0
+      sum by (node_prefix) (increase(leader_memory_pressure_rejections{node_prefix="$node_prefix"}[10m]))
+    +
+      sum by (node_prefix) (
+        increase(follower_memory_pressure_rejections{node_prefix="$node_prefix"}[10m])
+      )
+  +
+    sum by (node_prefix) (
+      increase(operation_memory_pressure_rejections{node_prefix="$node_prefix"}[10m])
+    )
+>
+  0
 ```
 
 #### DB version mismatch
 
 Version mismatch has been detected for universe `'$universe_name'` for `$value` Master or T-Server instances.
 
 ```promql
-ybp_health_check_tserver_version_mismatch{universe_uuid="$uuid"} + ybp_health_check_master_version_mismatch{universe_uuid="$uuid"} > 0
+   ybp_health_check_tserver_version_mismatch{universe_uuid="$uuid"}
```

Comment:
also looks a bit off

Reply:
same with this. Verified it again with the parser.